src/Playroom/Playroom.tsx

```diff
@@ -133,11 +133,18 @@ export default ({ components, themes, widths, snippets }: PlayroomProps) => {
     </Fragment>
   );
 
+  if (editorHeightPercentage < 0 || editorHeightPercentage > 100) {
+    throw new Error('editorHeightPercentage must be a value between 0 and 100');
+  }
+  if (editorWidthPercentage < 0 || editorWidthPercentage > 100) {
+    throw new Error('editorWidthPercentage must be a value between 0 and 100');
+  }
```

Comment:
If we are defaulting it when would this be needed? Only asking because we dont validate other properties this way, and if we [format the stored value to a percentage](https://github.com/seek-oss/playroom/pull/363/files#r1751023795) then this will require parsing.

Reply:
Ahh good point. I can't see it causing an issue, so probably not necessary. Have removed